### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Florian Chollet, Alexandre Delétra

### DISCUSSION THREAD

**Discussion by luma2010** [heig-vd-dai-course/heig-vd-dai-course#494] [Show and tell]
Notre idée pour ce deuxième projet et d'utiliser notre premier projet (le modificateur de variable "var=val").
Nous voudrions crée une application afin de permettre une connexion à un serveur avec quelques fonctions basic (ls, cd, etc...), et d'utiliser notre premiere application afin de modifier des fichier texts présent sur le serveur.
Pour la gestions de connexions simultané, nous avons décider de ne pas permettre plusieurs interactions sur le même fichier en même temps (avertir un utilisateur avec un message d'erreur/avertisement), mais que plusieurs utilisateurs puissent modifier plusieurs fichier en simultané.

**Comment by ludelafo** [comment]
OK pour nous !

**Comment by luma2010** [comment]
Voila notre rendu final :
https://github.com/[luma2010/heig-vd-dai-projet-pratique2](https://github.com/luma2010/heig-vd-dai-projet-pratique2)
Hash : f85f6a22a291b2576febfb5ab9ae17bf974daddb
